var/spack/repos/builtin/packages/cabana/package.py

```diff
@@ -23,14 +23,17 @@ class Cabana(CMakePackage):
     depends_on("cmake@3.9:", type='build')
     depends_on("kokkos+serial", when="+serial")
     depends_on("kokkos+openmp", when="+openmp")
+    depends_on("kokkos+cuda", when="+cuda")
 
     def cmake_args(self):
         options = [
             '-DCabana_ENABLE_TESTING=ON',
             '-DCabana_ENABLE_Serial=%s'  % (
                 'On' if '+serial'  in self.spec else 'Off'),
             '-DCabana_ENABLE_OpenMP=%s'  % (
-                'On' if '+openmp'  in self.spec else 'Off')
+                'On' if '+openmp'  in self.spec else 'Off'),
+            '-DCabana_ENABLE_Cuda=%s'  % (
+                'On' if '+cuda'  in self.spec else 'Off')
```

Comment:
The `Cabana` package doesn't currently have a `cuda` variant so I think this would always be false

Reply:
Fixed!